examples/playground/app/websites/page.tsx

```
@@ -0,0 +1,169 @@
+
+    try {
+      const { data, error } = await supabase.rpc('start_analyze_website_flow', {
+        url,
+      });
+
+      if (error) {
+        setFormError(error.message);
+        return;
+      }
+
+      if (data && data.run_id) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)